### ISSUE TITLE
Oculus VR support

### ISSUE THREAD

**Issue by janisozaur** [SFTtech/openage#402]
Hey, you should support Oculus and other VR headsets!

I would suggest using tapping into [project tango](https://www.google.com/atap/project-tango/) to generate necessary 3d data in realtime.

At some later stage you could add PS Move/Kinect/Wiimote compatibility and option to micromanage selected unit, so you could become gatherer, hunter or crossbowman…

**Comment by franciscod** [comment]
just do it, lol!

**Comment by liamdiprose** [comment]
Would this not mean buildings and units will have to be 3D models so you can veiw them at different angles? Unless you wanted town centres to look like cardboard cut-outs, which wouldn't be all-bad.

**Comment by TheJJ** [comment]
Yes, the depth effect with our current approach is limited to the terrain and flat object positioning, as the sprites have no depth information. If we are awesome, we can use modern computer vision to try to calculate the depth from the shadows and flat colors, which would look very nice but requires much effort and computing power.

Or somebody creates actual 3d models, but then again, we don't want to be 0ad and a 3d-game.

**Comment by lisacvuk** [comment]
I hate necroing old threads, but... If you plan on making your own sprites, you could just make rotations too, making it easy to add VR support. ... Just found this project, and now I'm typing this comment.

**Comment by franciscod** [comment]
You mean rotations for the buildings, for rotating the game viewport, original The Sims style?

**Comment by lisacvuk** [comment]
Yes, I guess.

**Comment by lisacvuk** [comment]
Or Doom sprites.

**Comment by Jogai** [comment]
> so you could become gatherer, hunter or crossbowman…
Yeah, make it an mmorpg

AR would be nice, playing it with friends, walking trough around the map in god-view..
